src/js/base/runtime.js

```diff
@@ -5211,7 +5211,7 @@ function (Namespace, jsnums, codePoint, util, exnStackParser, loader, seedrandom
       'raw-array-from-list': makeFunction(raw_array_from_list, "raw-array-from-list"),
       'raw-array-join-str': makeFunction(raw_array_join_str, "raw-array-join-str"),
       'get-value': makeFunction(getValue, "get-value"),
-      'list-to-raw-array': makeFunction(function(l) { return thisRuntime.ffi.toArray(l); }, "list-to-raw-array"),
+      'list-to-raw-array': makeFunction(raw_array_from_list, "raw-array-from-list"),
```

Comment:
I don't understand this line: I thought you were trying to get rid of this function entirely?  Why reimplement it, and change its name, if that's going to happen?

Reply:
The plot library current uses `list-to-raw-array` because `raw-array-from-list` in `master` is not usable (buggy). Once this is merged to `master`, we can get rid of it.